cinema_service/settings.py

```diff
@@ -125,14 +125,18 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = "media"
+
+MEDIA_URL = "/media/"
```

Comment:
The `MEDIA_ROOT` and `MEDIA_URL` settings are defined, which is good for handling media files. Ensure that the `MEDIA_ROOT` directory exists and is properly configured in your deployment environment to store uploaded files.